runtime/doc/change.txt

```diff
@@ -1195,7 +1195,7 @@ and ":put" commands and with CTRL-R.
 		"@:" to repeat the previous command-line command.
 		The command-line is only stored in this register when at least
 		one character of it was typed.  Thus it remains unchanged if
-		the command was completely from a mapping.
+		the command was completed from a mapping.
```

Comment:
It's not a typo, it is the counterpart to the previous sentence which mentions "at least one character".

Thanks for the help though!

Reply:
How about *'Thus it remains unchanged if the entire command was from a mapping.'*? Would that prevent the confusion while still keeping the meaning?

Reply:
Not sure that captures the intent precisely. When I'm not certain about a change I just leave it alone, let Bram maintain it. Perhaps send your change to https://github.com/vim/vim/ and discuss it there.